html/global_attributes.json

```diff
@@ -846,6 +846,53 @@
           }
         }
       },
+      "enterkeyhint": {
+        "__compat": {
+          "support": {
+            "chrome": {
+              "version_added": "77"
+            },
+            "chrome_android": {
+              "version_added": "77"
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": {
+              "version_added": false
+            },
```

Comment:
https://developer.apple.com/documentation/safari_release_notes/safari_13_1_beta_release_notes

This is in safari 👍

Reply:
Whoops. I thought I added it to all versions.